CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ Contributors:
 - Fixed an argument issue with the `create_schema` macro on bigquery ([#2445](https://github.com/fishtown-analytics/dbt/issues/2445), [#2448](https://github.com/fishtown-analytics/dbt/pull/2448))
 - dbt now logs using the adapter plugin's ideas about how relations should be displayed ([dbt-spark/#74](https://github.com/fishtown-analytics/dbt-spark/issues/74), [#2450](https://github.com/fishtown-analytics/dbt/pull/2450))
 - The create_adapter_plugin.py script creates a version 2 dbt_project.yml file ([#2451](https://github.com/fishtown-analytics/dbt/issues/2451), [#2455](https://github.com/fishtown-analytics/dbt/pull/2455))
+- Remove misleading "Opening a new connection" log message in set_connection_name. [#2511](https://github.com/fishtown-analytics/dbt/issues/2511)
```

Comment:
Can you please move this up into the 0.18.0 section above, and add yourself to the `Contributors:` link there, like the other entries? I think that's the last thing we need before I can kick off tests and we can get this merged.

Reply:
Done!